### PR TITLE
doc: update usage with the recent commands

### PR DESCRIPTION
**Pull request by igorshubovych**
Added examples of `--linux`, `--osx`, `--sunos` options.

@rubenvereecken, @waldyrious could you, guys, please take a look? I am not very good in English.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,7 +56,8 @@ This file has to be valid JSON:
 }
 ```
 
-If you need to always read pages for different platform (e.g. Linux), you can put it in config file:
+If you need to always read pages for different platform (e.g. Linux),
+you can put it in config file:
```
Comment (waldyrious) [inline README.md]: "for *a* different platform", "in *the* config file"

File: README.md [hunk 2]
```diff
@@ -21,7 +21,10 @@ $ npm install -g tldr
 To see tldr pages:
 
 - `tldr <command>` show examples for this command
-- `tldr <command> --os=linux` show page for Linux version of this command
+- `tldr <command> --os=<platform>` show command page for given platform (`linux`, `osx`, `sunos`)
```
Comment (waldyrious) [inline README.md]: "for *the* given platform"

**Comment by waldyrious** [comment]
LGTM, only minor comments added inline.

File: README.md (hunk 1 above)
Comment (rubenvereecken) [inline README.md]: While we're at it.. "If you regularly need pages for a different platform" or something like that, "to always read" won't work.

**Pull request merged**